### PR TITLE
feat: align backgrounds and uniform spacings COMPASS-6148

### PR DESCRIPTION
**Pull request by mcasimir**
### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/compass-schema/src/components/compass-schema/compass-schema.module.less
Comment (Anemy) [inline packages/compass-schema/src/components/compass-schema/compass-schema.module.less:102]:
```suggestion
```

File: packages/compass-schema/src/components/compass-schema/compass-schema.module.less
Comment (Anemy) [inline packages/compass-schema/src/components/compass-schema/compass-schema.module.less:119]:
```suggestion
```

**Review by Anemy — APPROVED** [review]
lgtm! Once tests and checks pass. Left two small comments on commented code to remove.

Also before merging, do we want to update the backgrounds of the `InteractivePopover` to white, and the aggregations settings background color?

Pipeline settings:


Popover before/after:
Screen Shot 2022-10-03 at 4 10 17 PM
Screen Shot 2022-10-03 at 4 11 04 PM

File: packages/compass-collection/src/components/collection/collection.tsx
Comment (Anemy) [inline packages/compass-collection/src/components/collection/collection.tsx:149]: As this `Collection` component is no longer the user of the `darkMode` prop, it's passed down, it's probably a bit cleaner to remove the property from this component and the `withTheme` wrapper and instead have the `withTheme` wrapper inside of `CollectionHeader.tsx`

**Comment by mcasimir** [comment]
> Also before merging, do we want to update the backgrounds of the InteractivePopover to white, and the aggregations settings background color?

That's a good point, actually doesn't look bad in white at all .. maybe with some adjustments to the components there .. would definitely need some more design input though. We can do that when we convert those components to LG

**Pull request merged**
